### DISCUSSION TITLE
你的專屬信箱小管家

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#747] [General]
不知道大家有沒有這種經驗
就是電子信箱被一堆你沒有興趣或不需要的廣告信件灌爆
對於擾人的信件若可以用新穎的方式回覆
應該是一件頗有趣的事情
（雖然是惡趣味哇哈哈
但還是分享給大家～
https://technews.tw/2022/12/31/autoresponder-email-robot-ellie-ai/

**Comment by alvin1226** [comment]
這看起來很方便ㄟ
